src/Tribe/Assets.php

```diff
@@ -181,7 +181,10 @@ public function filter_modify_to_module( $tag, $handle ) {
 		}
 
 		// Do not add Module to a Script that already has a type.
-		if ( false !== strpos( $tag, ' type=' ) ) {
+		if (
+			false !== strpos( $tag, ' type=' )
+			&& current_theme_supports( 'html5', 'script' ) // These themes already have the `type="text/javascript"` added by WordPress core.
+		) {
 			return $tag;
```

Comment:
```suggestion
               $replacement = 'type="module" ';
		return str_replace( 'type="text/javascript" ', $replacement, $tag );
```

maybe this?

Reply:
tbh, I'm not even sure we need the check for `current_theme_supports`.  if we do this `str_replace` only modules with text/javascript types will be replaced.

Modules without any type will still fall to the next return. I think this covers all of our bases

Reply:
[Because WP can be wiping out the type](https://github.com/WordPress/wordpress-develop/blob/8cc9e53847f8a46258d9e42c29f8d524ec0af592/src/wp-includes/script-loader.php#L2013), if the `current_theme_supports( 'html5', 'script' )`, and as a result if we get there the str_replace won't do anything because it won't find the `type="text/javascript"`